flink/v1.15/flink/src/main/java/org/apache/iceberg/flink/FlinkReadOptions.java

```diff
@@ -0,0 +1,108 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.iceberg.flink;
+
+import org.apache.flink.configuration.ConfigOption;
+import org.apache.flink.configuration.ConfigOptions;
+import org.apache.iceberg.TableProperties;
+import org.apache.iceberg.flink.source.StreamingStartingStrategy;
+
+/** Flink source read options */
+public class FlinkReadOptions {
+  private static final String PREFIX = "connector.iceberg.";
+
+  private FlinkReadOptions() {}
+
+  public static final ConfigOption<Long> SNAPSHOT_ID =
+      ConfigOptions.key("snapshot-id").longType().defaultValue(null);
+
+  public static final String CASE_SENSITIVE = "case-sensitive";
+  public static final boolean CASE_SENSITIVE_DEFAULT = false;
+  public static final ConfigOption<Boolean> CASE_SENSITIVE_OPTION =
+      ConfigOptions.key(PREFIX + CASE_SENSITIVE).booleanType().defaultValue(CASE_SENSITIVE_DEFAULT);
+
+  public static final ConfigOption<Long> AS_OF_TIMESTAMP =
+      ConfigOptions.key("as-of-timestamp").longType().defaultValue(null);
+
+  public static final String STARTING_STRATEGY = "starting-strategy";
+  public static final StreamingStartingStrategy STARTING_STRATEGY_DEFAULT =
+      StreamingStartingStrategy.INCREMENTAL_FROM_LATEST_SNAPSHOT;
+  public static final ConfigOption<StreamingStartingStrategy> STARTING_STRATEGY_OPTION =
+      ConfigOptions.key(PREFIX + STARTING_STRATEGY)
+          .enumType(StreamingStartingStrategy.class)
+          .defaultValue(STARTING_STRATEGY_DEFAULT);
+
+  public static final ConfigOption<Long> START_SNAPSHOT_TIMESTAMP =
+      ConfigOptions.key("start-snapshot-timestamp").longType().defaultValue(null);
+
+  public static final ConfigOption<Long> START_SNAPSHOT_ID =
+      ConfigOptions.key("start-snapshot-id").longType().defaultValue(null);
+
+  public static final ConfigOption<Long> END_SNAPSHOT_ID =
+      ConfigOptions.key("end-snapshot-id").longType().defaultValue(null);
+
+  public static final String SPLIT_SIZE = "split-size";
+  public static final long SPLIT_SIZE_DEFAULT = TableProperties.SPLIT_SIZE_DEFAULT;
```

Comment:
should we just use `TableProperties.SPLIT_SIZE_DEFAULT` directly? seems no benefit of clone the default value here.

Reply:
Make sence. Done. Thx.